### ISSUE TITLE
Remove ticketcount from reward actor params and logic

### ISSUE THREAD

**Issue by ZenGround0** [filecoin-project/specs-actors#289]
With rise of drand and fall of electionpost this no longer makes sense and is just wasted space in parameters.

**Comment by sternhenri** [comment]
Per my comment in the chat today @anorth, this should also include reverting how we compute reward back to ensure that large miners (i.e. with more than 1/e power) are properly incentivized. 

The full formula and rationale are [here](https://github.com/filecoin-project/specs/issues/603#issuecomment-548597481), with it previously [specc'd here](https://github.com/filecoin-project/specs/pull/610/files#diff-351a78e10f4cc4cce1862d27b75c8f0bR151). 

TLDR is miners with 1/e power are expected to win once at every round, which means that any power over 1/e of the network will not be accounted for unless the reward function accounts for minerPower. 

Not including this back would not be catastrophic, it simply means miners with > 1/e power (there should be few hopefully) will sybil.
